stdlib/private/SwiftReflectionTest/SwiftReflectionTest.swift

```diff
@@ -211,15 +211,15 @@ internal func sendBytes() {
   let count = Int(readUInt())
   debugLog("Parent requested \(count) bytes from \(address)")
   var totalBytesWritten = 0
-  var pointer = UnsafeMutableRawPointer(bitPattern: address)!
+  var pointer = UnsafeMutableRawPointer(bitPattern: address)
   while totalBytesWritten < count {
     let bytesWritten = Int(fwrite(pointer, 1, Int(count), stdout))
     fflush(stdout)
     if bytesWritten == 0 {
       printErrnoAndExit()
     }
     totalBytesWritten += bytesWritten
-    pointer = pointer.advanced(by: bytesWritten)
+    pointer = pointer?.advanced(by: bytesWritten)
```

Comment:
Will it be faster to force unwrap a pointer once, rather than what happens now?

Anyway, I don't think this refactoring should belong to this PR, which is simply to stop polluting the build log with warnings.